packages/reports/tests/integration/components/filter-builders/collapsed-test.ts

```diff
@@ -21,6 +21,7 @@ module('Integration | Component | filter-builders/collapsed', function(hooks) {
     this.set('displayName', 'Foo');
     this.set('filter', factory.createFilter('dimension', 'bardOne', 'Foo', {}, 'in', []));
     this.set('supportedOperators', [{ id: 'in', name: 'Equals', valuesComponent: 'mock/values-component' }]);
```

Comment:
Do we need this line now? 
```suggestion
```

Reply:
Deleted, we dont have any references without the searching for the selected operator